code/clue.js

```diff
@@ -48,12 +278,59 @@ const pickKiller = () => {
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
```

Comment:
I think the code will be easier to read if you remove the comments from the assignment. Maybe add your own comments if you want them for clarification?